### DISCUSSION TITLE
benefit problem

### DISCUSSION THREAD

**Discussion by nohla77** [community/community#60229] [GitHub Education]
### Hi everyone,

Hi
I applied for EDU github but for some reason I did not recieve any email neither I got the benefits even though I was acepted
I get this meesage (I provided the necessary official documents from my university with an official stamp)
Congrats! Your request for [@nohla77](https://github.com/nohla77) was approved and your benefits will be available soon.
what seems to be  the problem ?

**Comment by GhostOf0days** [comment]
It takes some time. Please wait 24 hours to 72 hours from the time you were accepted.

**Reply by GhostOf0days** [reply]
If you still don't receive them after that, please contact [Github Support](https://support.github.com/request).

**Comment by samus-aran** [comment]
Hi @nohla77 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
